electron/js/appInit.js

```diff
@@ -37,10 +37,14 @@ const fixUnityIcon = () => {
   }
 };
 
-const handlePortableFlag = () => {
-  if (argv.portable) {
+const handlePortableFlags = () => {
+  if (argv.portable || argv.user_data_dir) {
+    const directory = argv.user_data_dir ? argv.user_data_dir : 'Data';
+
     const EXEC_PATH = process.env.APPIMAGE || process.execPath;
-    const USER_PATH = path.join(EXEC_PATH, '..', 'Data');
+    const USER_PATH = path.join(EXEC_PATH, '..', directory);
```

Comment:
I think it is not going to work if an absolute path are given by the user.
It's going to generate currentDir/../<start of absolute path>